LEProc/Program.cs

```diff
@@ -88,7 +89,14 @@ private static void Main(string[] args)
 
             try
             {
-                Args = args;
+                var argWithQuotesList = Regex.Matches(Environment.CommandLine, @"[\""].+?[\""]|[^ ]+")
```

Comment:
Can you add a few comments here to explain the logic?

Reply:
This change is to keep original quotes in the command line. 

If we directly use args[] to make the command line, _cmd1.exe /c _cmd2.exe "path with space" will become _cmd1.exe /c _cmd2.exe path with space and will cause issues.

Reply:
Thanks! Could you write the sentence into the code?

Reply:
Sure. I have added comments to the code. Please review.

Reply:
LGTM!